forward/ft_headmodel_duneuro.m

```diff
@@ -0,0 +1,244 @@
+function headmodel = ft_headmodel_duneuro(mesh, varargin)
+
+% FT_HEADMODEL_DUNEURO creates a volume conduction model of the head
+% using the finite element method (FEM) for EEG and MEG. Different source models
+% are implemented, including the St. Venant, the subtraction and partial
+% integration model. This function takes as input a mesh with tetrahedral
+% or hexahedral elements and corresponding conductivities and returns
+% as output a volume conduction model which can be used to compute EEG/MEG
+% leadfields.
+%
+% Use as
+%   headmodel = ft_headmodel_duneuro(mesh,'conductivity', conductivities, ...)
+%   headmodel = ft_headmodel_duneuro(mesh,'grid_filename', grid_filename, 'tensors_filename', tensors_filename, ...)
+%
+% Required input arguments should be specified in key-value pairs and have
+% to include either
+%   grid_filename   = string, filename for grid
```

Comment:
it is unclear what these files are, how to create them, and which which file format these are to be specified.

Reply:
addressed here: 2b40b95